src/containers/Guide/GuideBlocks/index.tsx

```diff
@@ -1,107 +1,113 @@
+        provide in order to "unlock" or spend their coin balance. The method to unlock a balance lock uses balance keys,
+        which are provided within the message of every block.
+      </p>
+      <p>
+        Every message within a block is hashed, and that hash value will become the next balance lock. Because every
+        block contains a unique balance key (as created by the previous block), this ensures that for every block sent
+        over the network, the balance lock of the sender's account gets updated. So, banks and validators cannot process
+        the same block more than once. Here is the entire process in more details:
       </p>
 
       <DocImage alt="balance lock and key" maxWidth={1400} src={BalanceLockAndKey} />
 
       <p>
-        Note that there is one exception to the method in which balance locks are determined. The majority of the time
-        the hash value of the account owner's last sent block message determines this. This does, however, leave out the
+        There is one exception to the method in which balance locks are determined. Usually, the hash value of the
+        account owner's last sent block message determines the account's balance lock. This, however, leaves out the
         scenario in which the account owner must access their coin balance for the very first time before a block has
         ever been sent. This occurs when a separate user has sent funds to an account for the very first time, but
-        before that recipient has sent any coins (created any blocks) themselves. In this case, the balance lock for the
-        account will be set to the recipient's account number. Therefore, the rules of how balance locks are determined
-        can be simplified as follows:
+        before that recipient sends any coins (create any blocks) themselves. Here, the balance lock for the account
+        will be the recipient's account number.
       </p>
-
-      <DocList variant="ol">
+      <p>So, the following two rules outline how balance locks are determined:</p>
+      <DocList variant="ul">
         <li>If an owner has never sent coins before, the balance lock is equal to their account number.</li>
         <li>Otherwise, the balance lock is equal to the hash value of their most recently sent block's message.</li>
       </DocList>
 
       <p>
-        <strong>Important Note:</strong> The balance lock for an account is only updated when that account owner is
-        sending blocks. The balance "lock and key" system is similar to the lock and key for a mailbox. No mailbox key
-        is required when inserting mail into your mailbox. Only when opening the mailbox to access the contents inside
-        is when a key is required.
+        <div className="callout">
+          <strong>Important Note:</strong> The balance lock for an account is only updated when that account owner sends
```

Comment:
is there a distinction between `Important Note:` like shown here, and `Note:` shown in the other callout?

Reply:
I was planning to have a different color for the callout box for important notes, so was probably going to ask designers for specific colors. Originally, the text was Important note. It's an update in the colors I will do after I get a designer's eye on that